server/debug/examples/trace.rs

```diff
@@ -0,0 +1,169 @@
+extern crate debug;
+
+extern crate winapi;
+extern crate kernel32;
+
+use std::{env, mem, ptr};
+use std::collections::HashMap;
+use std::os::windows::io::AsRawHandle;
+
+fn main() {
+    let child = debug::Command::new(env::args().nth(1).unwrap())
+        .env_clear()
+        .debug()
+        .expect("failed to launch process");
+
+    let options = debug::SymbolHandler::get_options();
+    debug::SymbolHandler::set_options(winapi::SYMOPT_DEBUG | winapi::SYMOPT_LOAD_LINES | options);
+
+    let mut threads = HashMap::new();
+    let mut symbols = debug::SymbolHandler::initialize(child.as_raw_handle())
+        .expect("failed to initialize symbol handler");
+
+    let mut attached = false;
+
+    let mut done = false;
+    while !done {
+        let event = debug::DebugEvent::wait_event()
+            .expect("failed to get debug event");
+
+        let mut debug_continue = false;
+
+        use debug::DebugEventInfo::*;
```

Comment:
I usually do; this one is importing the members of an enum so I don't have to write `debug::DebugEventInfo::` before every match arm, without polluting the whole file with those names.